src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -918,58 +918,7 @@ public static Hashtable Synchronized(Hashtable table)
 
         public virtual void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
This gets used in a bunch of places and I think it might be able to leak out to user code. I'm not sure if removing ISerializable would be a breaking change.